hack/playground.html

```diff
@@ -45,7 +45,7 @@
 
     <!-- CSS Customization -->
     <link rel="stylesheet" href="assets/css/custom.css">
-    <link rel="stylesheet" href="assets/css/playground.css">
+    <link rel="stylesheet" href="assets/css/playground.css?ver=20170919">
```

Comment:
This forces browsers to ignore their cache.

Reply:
good idea